websites/Z/ZeroTwo/dist/metadata.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.5",
+  "service": "ZeroTwo",
+  "author": {
+    "name": "Connor!",
+    "id": "744603004493365330"
+  },
+  "category": "other",
+  "logo": "https://i.imgur.com/NmwMLaF.png",
+  "thumbnail": "https://i.imgur.com/Pu1DxbM.png",
+  "version": "1.0.0",
+  "color": "#ff40ff",
+  "tags": [
+    "bot",
+    "discord-bot",
+    "bot-discord",
+    "zerotwo-discord",
+    "zerotwo-discord-bot",
+    "zero-two",
+    "discord",
+    "bot-for-discord",
+    "zerotwo-bot-discord"
+  ],
+  "url": "zerotwo.wtf",
+  "settings": [
+    {
+      "id": "incognito",
+      "title": "Hide Presence",
```

Comment:
Why is this a setting if you can literally disable presences?

Reply:
read the first comment -  https://i.imgur.com/Zoh8A18.png

Reply:
That seems useless to me
Also you do realize github allows you to upload images right